src/main/java/com/github/junit5docker/Docker.java

```diff
@@ -20,6 +21,7 @@
 @Target({ElementType.TYPE})
 @Retention(RetentionPolicy.RUNTIME)
 @ExtendWith(DockerExtension.class)
+@Repeatable(value = Dockers.class)
```

Comment:
If reapeted, don't we risk to call the `DockerExtension` several times ? Would removing the `ExtendWith` fix the problem (since you added it to the `Dockers` annotation)?

Reply:
From the docs it seems that registering an extension a second time is fine, that the second attempt will be ignored.
http://junit.org/junit5/docs/current/user-guide/#extensions-registration-inheritance

By keeping it this way we allow users to still use the single Docker annotation by itself, without always having to wrap it in the Dockers annotation.

Reply:
Great ! Let's go for it then :)